addons/binding/org.openhab.binding.loxone/README.md

```diff
@@ -0,0 +1,125 @@
+# Loxone Binding
```

Comment:
I added examples of .items definitions, including one for the pushbutton.
As for the sitemaps and things, I need to take a closer look, since in my installation I don't use sitemaps but HABPanel. There should not be anything specific and not default though.